src/index.html

```diff
@@ -1,17 +1,175 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="field">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="Surname"
+            required
+          >
+        </label>
+
+        <label class="field">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="Name"
+            required
+          >
+        </label>
+
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            min="1"
+            max="100"
+            name="age"
+            required
+          >
+        </label>
+
+        <label class="field">
+          Full date of birth:
+          <input
+            type="date"
+            name="Birth"
+            required
+          >
+        </label>
+
+        <label class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="termsAndSome"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="field">
+          E-mail:
+          <input
+          type="email"
+          placeholder="email@example.com"
+          name="email"
+          required
+          >
```

Comment:
Pay attention on indents. Fix all similar places
```suggestion
          <input
            type="email"
            placeholder="email@example.com"
            name="email"
            required
          >
```